src/main/webapp/app/cancellation-approval-bottom-sheet/cancellation-approval-bottom-sheet.component.html

```diff
@@ -0,0 +1,10 @@
+<mat-action-list>
+  <button mat-list-item (click)="bottomSheetRef.dismiss(true)">
```

Comment:
also this tiny bit of logic would be nicer in the component ts file

Reply:
I did that intentionally. Could you tell me why is that really tiny thing must be put into .ts? Really curious.